docs/pages/repo/docs/core-concepts/monorepos/configuring-workspaces.mdx

```diff
@@ -0,0 +1,266 @@
+---
+title: Configuring Workspaces
+description: Turborepo lets you configure each workspace
+---
+
+import Callout from "../../../../../components/Callout";
+
+# Configuring Workspaces
+
+Most monorepos can declare a `turbo.json` in the root directory with a uniform
+[pipeline][2] that apply to all workspaces. Sometimes, a monorepo can contain
+workspaces that need to configure their tasks differently. To accommodate this,
+starting in version 1.8, Turborepo enables you to extend the root configuration
+with a `turbo.json` in any workspace. This flexibility enables a more diverse
+set of apps and packages to co-exist, and allows workspace owners to maintain
+specialized tasks and configuration without affecting other apps and packages of
+the monorepo.
```

Comment:
Do we need this bit `Most monorepos can declare a `turbo.json` in the root directory with a uniform[pipeline][2] that applies to all workspaces.` ? I think its cleaner to jump straight in like this:
```suggestion
From version 1.8 you can extend the root configuration of _any_ workspace with a `turbo.json` file. This flexibility enables a more diverse
set of apps and packages to co-exist, and allows workspace owners to maintain specialized tasks and configuration without affecting other apps and packages of the monorepo.
```